spectre-meltdown-checker.sh

```diff
@@ -560,6 +563,12 @@ is_cpu_affected()
 		amd_legacy_erratum "$(amd_model_range 0x17 0x30 0x0 0x4f 0xf)" && variant_zenbleed=vuln
 		amd_legacy_erratum "$(amd_model_range 0x17 0x60 0x0 0x7f 0xf)" && variant_zenbleed=vuln
 		amd_legacy_erratum "$(amd_model_range 0x17 0xa0 0x0 0xaf 0xf)" && variant_zenbleed=vuln
+
+		# Inception (Zen3 & Zen4)
+		# TODO: Should Zend2 & Zen1 be added. No ucode update will be released but they are affected too.
```

Comment:
https://www.kernel.org/doc/html/next/admin-guide/hw-vuln/srso.html#affected-processors those should be added as well.